### ISSUE TITLE
tags

### ISSUE THREAD

**Issue by ttodosi** [val-town/val-town-vscode#9]
hey having issues with tags for side bar
blobs don't seem to load either
is the idea that you have #tags in comments in a val and it will pick it up?

![Image](https://github.com/user-attachments/assets/13e6c64d-0640-4398-8b61-0f9b6a7db49c)
![Image](https://github.com/user-attachments/assets/ff1c5719-cc24-4665-a31d-20129c4b3ef4)

**Comment by pomdtr** [comment]
Yeah that was the intent. There might be some changes in the api response schema that we need to take account for.
